src/components/routes/LogViewerRoute.vue

```diff
@@ -196,11 +196,11 @@ export default {
         },
 
         toggleContext(k) {
-            this.$set(this.showContext, k, !this.showContext[k]);
+            this.showContext[k] ^= 1;
```

Comment:
sorry but this syntax is unreadable to me, can we please stick to boolean assignment?

Reply:
See bbd833f4c1f245d77bf4fa36d3bc5b5f7c0568ed